jobs/hyperkube_release.groovy

```diff
@@ -0,0 +1,33 @@
+// META
+repo = "coreos/kube-jenkins-config"
+
+// CONFIG
+org_whitelist = ['coreos', 'coreos-inc']
+job_admins = ['colemickens', 'ericchiang', 'rithujohn191', 'rphillips']
+user_whitelist = job_admins
+
+// JOBS
+job_name = "tku-2-release-hyperkube"
+
+pipelineJob(job_name) {
+  parameters {
+    stringParam('JENKINS_BRANCH', 'master', 'REQUIRED: repo containing Jenkinsfile')
+    stringParam('KUBERNETES_VERSION', 'origin/master', 'REQUIRED: git reference to build')
```

Comment:
Do we want to default to `origin/master` here? I guess this job will use the version info in the tree to determine the docker tag to use, and it won't hurt anything if someone just runs this against master. Though presumably they'd be running a tag through this job?

Reply:
master does seem better, but - yes - this would be a tag. I'll tweak that.